dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -429,6 +466,29 @@ public void manualPublish(final List<String> args) {
         }
     }
 
+    private DockstoreTool.RegistryEnum getRegistryEnum(String registry) {
+        for (DockstoreTool.RegistryEnum reg : DockstoreTool.RegistryEnum.values()) {
+            if (registry.equals(reg.name())) {
+                return reg;
+            }
+        }
+        return null;
+    }
+
+    private String getRegistryPath(String registry) {
+        for (Registry r  : Registry.values()) {
+            if (registry.equals(r.name())) {
+                if (r.hasCustomDockerPath()) {
+                    return null;
+                } else {
+                    return r.toString();
+                }
+            }
+        }
+
+        return null;
```

Comment:
Same here, also these two methods need Javadoc since we'll probably be re-using them heavily